### PR TITLE
Added Cypress tests for RDF rank setup page

### PR DESCRIPTION
**Pull request by mihailradkov**
Tested the building of the ranks with and without filtering.

Added several selectors in the rdf rank page for easier selections.

### CODE REVIEW

File: src/pages/rdfrank.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <link href="css/rdfrank.css" rel="stylesheet">
 <link href="js/lib/ng-tags-input/ng-tags-input.min.css" rel="stylesheet"/>
 
-<div class="page">
+<div id="rdf_rank" class="page">
```
Comment (svilenvelikov) [inline src/pages/rdfrank.html]: uhmm why the underscore?

File: test-cypress/integration/setup/rdf-rank.spec.js [hunk 2]
```diff
@@ -0,0 +1,211 @@
+describe('Setup / Autocomplete', () => {
```
Comment (desislava-hristova-ontotext) [inline test-cypress/integration/setup/rdf-rank.spec.js]: Copy paste from autocomplete?

Comment (mihailradkov) [reply test-cypress/integration/setup/rdf-rank.spec.js]: Yes. Will fix it.

File: src/pages/rdfrank.html (hunk 1 above)
Comment (mihailradkov) [reply src/pages/rdfrank.html]: No reason really. I usually write IDs with underscore while classes with dashes

Comment (mihailradkov) [reply src/pages/rdfrank.html]: Will change it to a dash

Comment (svilenvelikov) [reply src/pages/rdfrank.html]: Usually classess are with dash and ids are camelcase

Comment (mihailradkov) [reply src/pages/rdfrank.html]: Changed to camelcase

File: test-cypress/integration/setup/rdf-rank.spec.js (hunk 2 above)
Comment (mihailradkov) [reply test-cypress/integration/setup/rdf-rank.spec.js]: Fixed

**Pull request merged**